packages/api/package.json

```diff
@@ -13,6 +14,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "nodemon": "^2.0.4",
```

Comment:
What do you think of making this a dev dependency, since we're only using it for development/not in the code?

Reply:
Ahh I see. I just pushed an update to the PR.